src/vlinder/appreciate.py

```diff
@@ -181,3 +182,18 @@ def _apply_scenario_weights(self) -> None:
                 appreciation = self.output_dict[scenario][option]["decision_makers_option_appreciation"]
                 weighted_appreciation = appreciation * weight / total_weight
                 self.output_dict[scenario][option]["scenario_appreciations"] = weighted_appreciation
+
+    def _best_dmo_calculator(self) -> None:
+        """
+        This function calculates the sum of all weighted appreciations per DMO and return the highest sum
```

Comment:
"return" -> "returns"
:return: -> er wordt niks gereturned, net zoals in bijv. `appreciate_single_scenario`, dus: 
:return: None as results are stored within the output_dict